task_1_6.py

```diff
@@ -0,0 +1,13 @@
+# Напишите программу вычисления арифметического выражения заданного строкой. Используйте операции +,-,/,*. приоритет операций стандартный.Добавьте возможность использования скобок, меняющих приоритет операций.
+import re
+import os
+clear = lambda: os.system('cls')
+clear()
```

Comment:
Можно было сделать сразу так. Все равно вы ее вызывает единожды.
```suggestion
os.system('cls')
```
Кстати, это сработает только на Windows. На остальных платформах выйдет ошибка.